mirage/views/utils/-private.ts

```diff
@@ -218,15 +219,32 @@ export function compareBooleans(
     }
 }
 
+export function compareIds(
+    actualValue: any,
+    comparisonValue: string,
+    operator: ComparisonOperators,
+): boolean {
+    switch (operator) {
+    case ComparisonOperators.Eq:
+        return comparisonValue.split(',').includes(actualValue);
+    case ComparisonOperators.Ne:
+        return !comparisonValue.split(',').includes(actualValue);
+    default:
+        throw new Error(`IDs can't be compared with "${operator}".`);
+    }
+}
+
 export function compare(
-    actualValue: string | boolean,
+    actualValue: any,
     comparisonValue: string,
     operator: ComparisonOperators,
 ): boolean {
     if (typeof actualValue === 'string') {
         return compareStrings(actualValue, comparisonValue, operator);
     } else if (typeof actualValue === 'boolean') {
         return compareBooleans(actualValue, queryParamIsTruthy(comparisonValue), operator);
+    } else if (actualValue instanceof Array) {
+        return actualValue.includes(comparisonValue);
```

Comment:
`currentUserPermissions` on a `node` model instance is an array of strings. Currently our mirage backend doesn't allow filtering on this field because the `compare()` method is not implemented for such comparison.